openapi/spec/openapi.json

```diff
@@ -2611,6 +2645,33 @@
                 }
               }
             }
+          },
+          "failures": {
+            "type": "object",
+            "items": {
```

Comment:
Is `failures` an object or an array? `type: 'object'` and `items` are conflicting. The schema also doesn't seem to match the example above? I feel like these two lines, and one `}` down below, should be removed?